src/test/scala/me/rexim/morganey/interpreter/MorganeyCompilerSpec.scala

```diff
@@ -27,21 +25,16 @@ class MorganeyCompilerSpec extends FlatSpec with Matchers {
 
   val programInput = "khooy".toStream
 
-  "Compiler" should "compile a correct program to a reducible term" in {
+  behavior of "Morganey Compiler"
+
+  it should "compile a correct program to a reducible term" in {
     val Right(expectedProgram) = programBody.addBindings(MorganeyBinding(LambdaVar("input"), LambdaInput(() => programInput)) :: programBindings)
     MorganeyCompiler.compileProgram(() => programInput)(rawProgram) should be (Success(expectedProgram))
   }
 
-  "Compiler" should "fail an incorrect program" in {
+  it should "fail an incorrect program" in {
     MorganeyCompiler.compileProgram(() =>programInput)(programBindings).isFailure should be (true)
   }
 
-  "Compiler" should "not interpret empty loading nodes and just ignore them" in {
-    MorganeyCompiler.interpretNode(MorganeyLoading(None), moduleFinder, Set()) should be (Success(List()))
-  }
-
-  "Compiler" should "should returns bindings as is during their interpretation as a node" in {
-    val binding = MorganeyBinding(lvar("khooy"), lvar("khooy"))
-    MorganeyCompiler.interpretNode(binding, moduleFinder, Set()) should be (Success(List(binding)))
-  }
+  it should ""
```

Comment:
Haha! Nice one! :D Yeah, I think it's a leftover of some sort. Thanks for pointing that out. I'll remove it.